samples/HttpClientCanary/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.canary</groupId>
+  <artifactId>canary</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>software.amazon.awssdk.crt</groupId>
+      <artifactId>aws-crt</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
```

Comment:
This version doesn't work for me on an M1. I get the following error
```Unable to unpack AWS CRT lib: java.io.IOException: Unable to open library in jar for AWS CRT: /osx/x86_64/libaws-crt-jni.dylib
java.io.IOException: Unable to open library in jar for AWS CRT: /osx/x86_64/libaws-crt-jni.dylib
        at software.amazon.awssdk.crt.CRT.extractAndLoadLibrary(CRT.java:190)
        at software.amazon.awssdk.crt.CRT.loadLibraryFromJar(CRT.java:242)
        at software.amazon.awssdk.crt.CRT.<clinit>(CRT.java:34)
        at software.amazon.awssdk.crt.CrtResource.<clinit>(CrtResource.java:104)
        at com.canary.Http2StreamManagerCanary.createStreamManager(Http2StreamManagerCanary.java:51)
        at com.canary.Http2StreamManagerCanary.runCanary(Http2StreamManagerCanary.java:174)
        at com.canary.Http2StreamManagerCanary.main(Http2StreamManagerCanary.java:195)
Exception in thread "main" java.lang.ExceptionInInitializerError
        at software.amazon.awssdk.crt.CrtResource.<clinit>(CrtResource.java:104)
        at com.canary.Http2StreamManagerCanary.createStreamManager(Http2StreamManagerCanary.java:51)
        at com.canary.Http2StreamManagerCanary.runCanary(Http2StreamManagerCanary.java:174)
        at com.canary.Http2StreamManagerCanary.main(Http2StreamManagerCanary.java:195)
Caused by: software.amazon.awssdk.crt.CrtRuntimeException: software.amazon.awssdk.crt.CrtRuntimeException: Unable to unpack AWS CRT library UNKNOWN(-1) UNKNOWN(-1)
        at software.amazon.awssdk.crt.CRT.loadLibraryFromJar(CRT.java:252)
        at software.amazon.awssdk.crt.CRT.<clinit>(CRT.java:34)
        ... 4 more
```
        
I changed it to 0.19.3 and it is working now.

Reply:
Well, you need `mvn install on the root directory` first.

Reply:
Wait, that seems like weird...
It's search for `x86_64` on your arm machine.

Reply:
What `JAVA_HOME` you set? Do you have a x86 java on you machine?

Reply:
My Java Home is /Users/waqarak/Library/Java/JavaVirtualMachines/corretto-11.0.16/Contents/Home. I did run mvn install on the root directory.